api/common/v1alpha1/ref.go

```diff
@@ -55,6 +55,15 @@ func (k TargetRefKind) IsRealResource() bool {
 	}
 }
 
+func (k TargetRefKind) IsOldKind() bool {
```

Comment:
nit: please leave a doc comment for this method. Like these are kinds that's possible to use in Kuma policies before introducing support for real resources (i.e. MeshExternalService, MeshMutliZoneService and MeshService)